wandb/sdk/artifacts/artifact_manifest_entry.py

```diff
@@ -150,8 +150,13 @@ def download(
 
         # Skip checking the cache (and possibly downloading) if the file already exists
         # and has the digest we're expecting.
-        if os.path.exists(dest_path) and self.digest == md5_file_b64(dest_path):
-            return FilePathStr(dest_path)
+        try:
+            md5_hash = md5_file_b64(dest_path)
+        except (FileNotFoundError, IsADirectoryError):
+            pass
```

Comment:
is it ok to silently continue do you want to log it somewhere?

Reply:
added a debug log here!